debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/ThreadsCommand.java

```diff
@@ -216,10 +216,11 @@ private void flags(PrintStream out) throws DDRInteractiveCommandException {
 				J9VMThreadPointer threadCursor = vm.mainThread();
 
 				do {
-					out.append(String.format("    !j9vmthread 0x%s publicFlags=%s privateFlags=%s // %s", 
+					out.append(String.format("    !j9vmthread 0x%s publicFlags=%s privateFlags=%s inNative=%d // %s", 
 							Long.toHexString(threadCursor.getAddress()),
 							Long.toHexString(threadCursor.publicFlags().longValue()),
 							Long.toHexString(threadCursor.privateFlags().longValue()),
+							Long.toHexString(threadCursor.inNative().longValue()),
```

Comment:
`inNative` is a boolean value (i.e. 0 or 1), so the use of `Long.toHexString` isn't helpful.

Reply:
I left it in there in case it becomes something more - it still prints only 0 or 1.